lm-simulator/lm_simulator/schemas.py

```diff
@@ -22,19 +22,12 @@ class LicenseCreate(BaseModel):
 
 
 class LicenseRow(LicenseCreate):
-    """Model for the license in the database, we calculate the in_use value dynamically."""
-
-    id: Optional[int] = None
+    in_use: int = 0
     licenses_in_use: List[LicenseInUseRow] = []
-    # This is used to calculate the in_use value using the licenses_in_use field.
-    in_use: Optional[int] = 0
+
+    model_config = ConfigDict(from_attributes=True)
 
     @model_validator(mode="after")
-    def in_use_validator(self) -> int:
-        """This validator is used to calculate the in_use value."""
-        self.in_use = 0
-        for license_in_use in self.licenses_in_use:
-            self.in_use += license_in_use.quantity
+    def calculate_in_use(self):
+        self.in_use = sum(license_in_use.quantity for license_in_use in self.licenses_in_use)
```

Comment:
Won't this already be computed by the property `in_use` on the db model? Why re-implement here?

Reply:
That's a good point, I thought that the calculation had to happen in the schema as well.